task/pong/Cargo.toml

```diff
@@ -3,11 +3,12 @@ name = "task-pong"
 version = "0.1.0"
 edition = "2018"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+panic-messages = ["userlib/panic-messages"]
```

Comment:
It does change that, yes. We could edit all callers; I haven't gone ahead and done that because... well, okay, the most proximal reason is likely laziness, but my thought process was

- pong doesn't really panic.
- pong is a dev demo that isn't in production.

I can be talked out of this.

Reply:
Alright, I don't feel strongly about it!

Reply:
Frankly we kind of need better app-level controls for this sort of thing, so that making something like this configurable doesn't require running around adding a bazillion `features = ["panic-messages"]` to every app.toml. I'm not sure what that looks like yet though. Very open to suggestions.